### PR TITLE
forward call

### PR DESCRIPTION
**Pull request by troft**

### CODE REVIEW

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-forwardCall.s3-website-us-east-1.amazonaws.com/

File: bxml/buildingBlocks/createCall.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Create an Outbound Call
```
Comment (dtolb) [inline bxml/buildingBlocks/createCall.md]: Are we tracking create call in this PR or in the [PR #102](https://github.com/Bandwidth/ap-docs/pull/102)

File: bxml/buildingBlocks/forwardCall.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+## Concept
+
+To forward a call to another number, we will use the transfer BXML verb.
+
+## Use Cases
+
+| Use Case                                    | BXML Code                                                 |
+|:--------------------------------------------|:----------------------------------------------------------|
+| Call centers: A customer can call in and a recording will play. The call will then be transfered to the first available tenant. | Add multiple phone numbers(`<PhoneNumber>+15552221234</PhoneNumber>`) |
+| Appointment reminders: If a customer would like to reschedule, you can transfer their call to the scheduling office | the code will be the same as the example|
```
Comment (dtolb) [inline bxml/buildingBlocks/forwardCall.md]: `the code will be the same as the example`

Capital `T`

`The code will be the same as the example`

File: bxml/buildingBlocks/forwardCall.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+| Operator: Often times customers can press 0 to be connected to an operator. | The code would need to collect digits then transfer to an operator if 0 is pushed|
```
Comment (dtolb) [inline bxml/buildingBlocks/forwardCall.md]: Add link to how to collect digits

**Comment by dtolb** [comment]
The `use-cases` section is genius

**Comment by dtolb** [comment]
I think adding a `BXML Verbs Used` section and links to the full documentation would also be good addition

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-forwardCall.s3-website-us-east-1.amazonaws.com/

File: bxml/buildingBlocks/createCall.md (hunk 1 above)
Comment (troft) [reply bxml/buildingBlocks/createCall.md]: They are different

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-forwardCall.s3-website-us-east-1.amazonaws.com/

File: bxml/buildingBlocks/createCall.md (hunk 1 above)
Comment (dtolb) [reply bxml/buildingBlocks/createCall.md]: Looks like this file is the same one in the [create call](https://github.com/Bandwidth/ap-docs/pull/102/files#diff-ba54ee7b02e978bb14411df3d46bf659) pull request.

File: bxml/buildingBlocks/forwardCall.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Forward a Call
+
+## Concept
+
+To forward a call to another number, we will use the transfer BXML verb.
```
Comment (dtolb) [inline bxml/buildingBlocks/forwardCall.md]: `To forward a call to another number, we will use the [`<Transfer>`](../verbs/transfer.md) BXML verb.`

Comment (dtolb) [reply bxml/buildingBlocks/forwardCall.md]: Add links where possible

**Comment by dtolb** [comment]
Preview Changes at:
        http://bw-apdocs-forwardCall.s3-website-us-east-1.amazonaws.com/

**Comment by dtolb** [comment]
Looks good!

**Pull request merged**